janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchSetup.java

```diff
@@ -262,25 +272,17 @@ static void applySettingsFromJanusGraphConf(ImmutableSettings.Builder settings,
     }
 
 
-    private static void makeLocalDirsIfNecessary(ImmutableSettings.Builder settingsBuilder, Configuration config) {
+    private static void makeLocalDirsIfNecessary(Settings.Builder settingsBuilder, Configuration config) {
         if (config.has(INDEX_DIRECTORY)) {
             String dataDirectory = config.get(INDEX_DIRECTORY);
             File f = new File(dataDirectory);
             if (!f.exists()) {
                 log.info("Creating ES directory prefix: {}", f);
                 f.mkdirs();
             }
-            for (String sub : ElasticSearchIndex.DATA_SUBDIRS) {
-                String subdir = dataDirectory + File.separator + sub;
-                f = new File(subdir);
-                if (!f.exists()) {
-                    log.info("Creating ES {} directory: {}", sub, f);
-                    f.mkdirs();
-                }
-                settingsBuilder.put("path." + sub, subdir);
-                log.debug("Set ES {} directory: {}", sub, f);
-            }
+            settingsBuilder.put("path.home", dataDirectory);
```

Comment:
ive seen this property key before. can we collect all ES property keys someplace and use string constants?